### ISSUE TITLE
Change duration format

### ISSUE THREAD

**Issue by rhuanbarreto** [jsGanttImproved/jsgantt-improved#333]
**Describe the solution you'd like**
I want to display the duration in the column in hours instead of days.

**Comment by mariohmol** [comment]
I think this is dup to https://github.com/jsGanttImproved/jsgantt-improved/issues/253#issuecomment-529994760 right?

**Comment by rhuanbarreto** [comment]
No. I don't want an hour view. I want the duration on the column to be shown in hours instead of days, weeks or months depending on the view mode.

**Comment by mariohmol** [comment]
Makes sense, o in the day view, when the duration is less then a day, show the duration in hours.

is this correct?

**Comment by rhuanbarreto** [comment]
I want to have a fixed duration format independent of view. In my case, all tasks are counted in hours, so in day, week, month and quarter view, the duration will display the value I sent in hours.

**Comment by mariohmol** [comment]
I see.. well.. this should not be hard to implement

1. create a new option for the gantchart will include a durationFormat.
https://github.com/jsGanttImproved/jsgantt-improved/blob/master/src/options.ts#L77 

2. You can make a logic here using this durationFormat to change how it calculates:
https://github.com/jsGanttImproved/jsgantt-improved/blob/master/src/task.ts#L249

3. include the hour format here
https://github.com/jsGanttImproved/jsgantt-improved/blob/master/src/task.ts#L263
